THIRD-PARTY

```diff
@@ -1,39 +1,37 @@
 
-Dependency License Report for opensearch-data-prepper 1.4.0-SNAPSHOT
+Dependency License Report for opensearch-data-prepper 1.5.0-SNAPSHOT
 
-1. Group: com.amazonaws  Name: aws-java-sdk-acm  Version: 1.12.155
+1. Group: com.aayushatharva.brotli4j  Name: brotli4j  Version: 1.7.1
```

Comment:
This seems like an odd change, is this supposed to be accurate?

Reply:
The order of dependencies is not consistent, so you can't rely on a line-by-line change. 

Also, the `aws-java-sdk-acm` dependency was removed entirely in this version.